src/constants/env.ts

```diff
@@ -0,0 +1 @@
+export const URL_PRODUCT_API = process.env.NEXT_PUBLIC_URL_PRODUCT_API || ''
```

Comment:
As I understood, you propose to create constants folder instead of using constants.ts file in utils?

Reply:
Yes, there can be a lot of constants some dag

Reply:
Ok